autokeras/hypermodels/heads.py

```diff
@@ -169,3 +170,102 @@ def build(self, hp, inputs=None):
 
     def get_adapter(self):
         return adapters.RegressionHeadAdapter(name=self.name)
+
+
+class SegmenterHead(head_module.Head):
+    """Segmentation Dense layers.
+
+    Use sigmoid and binary crossentropy for binary classification and multi-label
+    classification. Use softmax and categorical crossentropy for multi-class
+    (more than 2) segmentation. Use Accuracy as metrics by default.
+
+    The targets passing to the head would have to be tf.data.Dataset, np.ndarray,
+    pd.DataFrame or pd.Series. It can be raw labels, one-hot encoded if more than two
+    classes, or binary encoded for binary classification.
+
+    The raw labels will be encoded to one column if two classes were found,
+    or one-hot encoded if more than two classes were found.
+
+    # Arguments
+        num_classes: Int. Defaults to None. If None, it will be inferred from the
+            data.
+        multi_label: Boolean. Defaults to False.
```

Comment:
We don't support multi_label for the segmentation for now. Please remove this argument